src/i18n/en-US.json

```diff
@@ -1000,6 +1000,8 @@
   "preferencesAccountPrivacy": "Privacy",
   "preferencesAccountReadReceiptsCheckbox": "Read receipts",
   "preferencesAccountReadReceiptsDetail": "When this is off, you won’t be able to see read receipts from other people. This setting does not apply to group conversations.",
+  "preferencesAccountTypingIndicatorCheckbox": "Typing Indicator",
+  "preferencesAccountTypingIndicatorsDetail": "When this is off, you won’t be able to see when other people are typing, and others won’t see when you are typing. This setting applies to all conversations on this device.",
```

Comment:
Is the "...on this device" part enough or should it say something along the lines of "... on all devices using the web or desktop client" ? It sounds that the api would make the setting shared across devices using the web client no?

Reply:
Well for now it's not implemented on mobile devices. I guess we could change this wording when mobile devices implement it (but you are correct, it's sync with web devices)